mk/spksrc.common.mk

```diff
@@ -23,52 +23,41 @@ PIP_WHEEL = $(PIP) $(PIP_WHEEL_ARGS)
 # Available languages
 LANGUAGES = chs cht csy dan enu fre ger hun ita jpn krn nld nor plk ptb ptg rus spn sve trk
 
-# Toolchains
-AVAILABLE_TCS = $(notdir $(wildcard ../../toolchains/syno-*))
-AVAILABLE_ARCHS = $(notdir $(subst syno-,/,$(AVAILABLE_TCS)))
-
-# Toolchain filters
-SUPPORTED_ARCHS = $(sort $(filter-out $(ARCHS_DUPES), $(AVAILABLE_ARCHS)) apollolake-6.2.3 geminilake-6.2.3 purley-6.2.3)
```

Comment:
My intent here with the trailing `apollolake-6.2.3 geminilake-6.2.3 purley-6.2.3` was to change this into `FORCED_ARCH` or something similar so when I compile lets say `ffmpeg` I could force some specific archs at `all-supported` time to create various x86_64 packages so `libdrm` would enable either Intel OR AMD and depending of   `TC_KERNEL` version I can then either only compile the old intel driver or both on newer 4.4+ platforms.

So below at line 54 when defining `SUPPORTED_ARCHS` I would include room for a added variable such as `FORCED_ARCHS` to be appended to the resulting value.

Reply:
And my intention is that `make all-supported` shall build the default TC version (or the highest TC version of the arch if lower) of all supported archs.
`make all-supported` is the minimal amount of packages every contributor must build to validate a pull-request.

Reply:
Besides the `apollolake-6.2.3 geminilake-6.2.3 purley-6.2.3` my previous changes the intent was that `all-supported` output be closer to github-action.  Now that would be different and move back to the previous behavior if I understand correctly?

As for the `FORCED_ARCH` I'll look into that later once this PR is merged.